pyproject.toml

```diff
@@ -54,10 +54,11 @@ paho-mqtt = ">=1.6,<3"
 python-gnupg = ">=0.4.8,<0.6.0"
 
 [tool.poetry.dev-dependencies]
-pylint = "^3.2.5"
+pylint = "^3.2.6"
```

Comment:
we should either use pylint **or** ruff ... also: `pylint` should not be required to be a direct import anyway!

You may want to remove pylint related import(s)!